include/os/android.h

```diff
@@ -0,0 +1,14 @@
+/* android.h - A secure memory allocator
+ * Copyright 2022 - chris.rohlf@gmail.com */
+
+#pragma once
+
+#include <sys/prctl.h>
+
+#ifndef PR_SET_VMA
+#define PR_SET_VMA 0x53564d41
```

Comment:
What the hell is this magic number :')
It should be documented.

Reply:
Not my magic number but we can document it https://android.googlesource.com/platform/bionic/+/263325d/libc/include/sys/prctl.h#42

Reply:
I hate computers.